.travis.yml

```diff
@@ -8,5 +8,9 @@ node_js:
   - "5"
   - "4"
   - "4.2"
+env:
+  - HUBOT_VERSION=">=2.0 <3"
+  - HUBOT_VERSION="^3.0"
 after_success:
   - npm run codecov
+script: npm install && npm install hubot@"$HUBOT_VERSION" && npm test
```

Comment:
doh, that makes total sense. thanks for walking me through it. maybe we can add a comment here so that future maintainers have some indication of how this works. something along the lines of "explicitly install hubot with an overridden version so that we can check for issues before the latest major version as seen in package.json". that might be too wordy, but something like that would be good.

Reply:
OK, pushed with a comment!